boards/arm/sama5/sama5d3-xplained/Kconfig

```diff
@@ -113,8 +113,7 @@ endchoice # AT25 serial FLASH configuration
 
 config SAMA5D3XPLAINED_USBHOST_STACKSIZE
 	int "USB host waiter stack size"
-	default 1536 if USBHOST_HUB
-	default 1024 if !USBHOST_HUB
```

Comment:
It is inappropriate to increase the size of this tuned stack size.  Let's not do that.